app/assets/javascripts/services/miq_service.js

```diff
@@ -40,6 +40,13 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', '$log
     miqSparkleOff();
   };
 
+  this.replacePartials = function(data) {
+    var partials = data.replacePartials
+    $.each(partials, function(elementId, content) {
```

Comment:
I wouldn't use `$.each`, but rather default `forEach`, it does not messes up with `this` object. And you can get rid of the variable assignement as well.
```javascript
Object.keys(data.replacePartials).forEach(function(elementId) {
  $('#' + elementId).replaceWith(data.replacePartials[key]);
});
```